### PR TITLE
Datadog apm tracing

### PR DESCRIPTION
**Pull request by hope-yu-fivestars**

### CODE REVIEW

File: app/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
+ddtrace==0.10.0
```
Comment (agianey) [inline app/requirements.txt]: 0.11.0 is out

**Comment by agianey** [comment]
Looks fine, but there are some things that are Fivestars specific, like the nodeselector stuff, that I don't think we'd want in a "public tutorial" of aladdin, but the tradeoff of removing it is making it slightly more cumbersome for other fivestars projects to adopt this pattern, so I'm not sure what the right answer is.

File: app/requirements.txt (hunk 1 above)
Comment (hope-yu-fivestars) [reply app/requirements.txt]: 👍

**Comment by hope-yu-fivestars** [comment]
I think for now since aladdin isn't open sourced yet, the main audience of this is going to be Fivestars and we should prioritize ease of use for that. As long as there isn't sensitive information included, we can always go back and refactor it to be more generally applicable.

**Review by agianey — APPROVED** [review]

File: app/entrypoint.sh [hunk 2]
```diff
@@ -1,3 +1,10 @@
 #!/bin/sh
 
-uwsgi /config/uwsgi.yaml
+if $APM_ENABLED; then
+    echo "APM tracing enabled: $APM_APP_NAME"
```
Comment (hope-yu-fivestars) [inline app/entrypoint.sh:4]: Small typo here that was fixed in a later pr, APM_APP_NAME should be DATADOG_SERVICE_NAME.

**Pull request merged**